scripts/build/termux_step_start_build.sh

```diff
@@ -29,18 +29,29 @@ termux_step_start_build() {
 		DEBUG=""
 	fi
 
-	if [ -z "$TERMUX_DEBUG" ] &&
-	   [ -z "${TERMUX_FORCE_BUILD+x}" ] &&
-	   [ -e "/data/data/.built-packages/$TERMUX_PKG_NAME" ]; then
-		if [ "$(cat "/data/data/.built-packages/$TERMUX_PKG_NAME")" = "$TERMUX_PKG_FULLVERSION" ]; then
-			echo "$TERMUX_PKG_NAME@$TERMUX_PKG_FULLVERSION built - skipping (rm /data/data/.built-packages/$TERMUX_PKG_NAME to force rebuild)"
-			exit 0
+	if [ -z "$TERMUX_DEBUG" ] && [ -z "${TERMUX_FORCE_BUILD+x}" ]; then
+		if [ -n "$TERMUX_ON_DEVICE_BUILD" ]; then
+			if [ -e "$TERMUX_TOPDIR/.built-packages/$TERMUX_PKG_NAME" ] &&
```

Comment:
What about all the packages we manually install above? They won't be listed in here. Shouldn't there also be a dpkg-query here to check if the package is installed?

Reply:
Done.